### PR TITLE
WIP: integrate nextclade doc pages in docs.nextstrain.org

### PR DESCRIPTION
**Pull request by rneher**
This is still work in progress. This branch currently pulls `.md` files from the nexclade branch `docs/rtd`.

### CODE REVIEW

File: src/fetch-docs.py [hunk 1]
```diff
@@ -3,6 +3,9 @@
 import requests
 from pathlib import Path
 
+nextclade_branch = 'docs/rtd'
```
Comment (eharkins) [inline src/fetch-docs.py:6]:
```suggestion
nextclade_branch = 'master'
```

File: src/fetch-docs.py [hunk 2]
```diff
@@ -44,6 +47,11 @@
     f'{ncov_url}interpretation.md': 'tutorials/SARS-CoV-2/steps/interpretation.md',
     f'{ncov_url}narratives.md': 'tutorials/SARS-CoV-2/steps/narratives.md',
     f'{ncov_url}naming_clades.md': 'tutorials/SARS-CoV-2/steps/naming_clades.md',
+    f'{nextclade_url}user/nextclade-algorithm.md': 'learn/nextclade.md',
```
Comment (eharkins) [inline src/fetch-docs.py:50]: Was this ([and the link below](https://github.com/nextstrain/docs.nextstrain.org/pull/62/files#diff-f0af9e6baf0df1c1f6c46ad21e9ca0d9bc1e9d35a202b33a0fe3751cfbd89535R16)) trying to include the whole https://docs.nextstrain.org/projects/nextclade/en/latest/user/algorithm/index.html directory/section? I think you need to get all of those files from that directory and then reference the index if so. However, I think you could just link to the nextclade docs in this case.

File: src/fetch-docs.py [hunk 3]
```diff
@@ -44,6 +47,11 @@
...
+    f'{nextclade_url}user/nextalign-cli-flags.md': 'reference/nextalign-cli.md',
```
Comment (eharkins) [inline src/fetch-docs.py:51]:
```suggestion
    f'{nextclade_url}user/nextalign-cli.md': 'reference/nextalign-cli.md',
```

File: src/fetch-docs.py [hunk 4]
```diff
@@ -44,6 +47,11 @@
...
+    f'{nextclade_url}user/nextclade-cli-flags.md': 'reference/nextclade-cli.md',
```
Comment (eharkins) [inline src/fetch-docs.py:52]:
```suggestion
    f'{nextclade_url}user/nextclade-cli.md': 'reference/nextclade-cli.md',
```

File: src/fetch-docs.py [hunk 5]
```diff
@@ -44,6 +47,11 @@
...
+    f'{nextclade_url}user/nextclade-cli-tutorial.md': 'tutorials/nextclade-cli-tutorial.md',
```
Comment (eharkins) [inline src/fetch-docs.py:53]:
```suggestion
```

File: src/fetch-docs.py [hunk 6]
```diff
@@ -44,6 +47,11 @@
...
+    f'{nextclade_url}user/nextclade-web.md': 'tutorials/nextclade-web-tutorial.md',
```
Comment (eharkins) [inline src/fetch-docs.py:54]:
```suggestion
    f'{nextclade_url}user/nextclade-web.md': 'tutorials/nextclade-web.md',
```

File: src/tutorials/index.rst [hunk 7]
```diff
@@ -15,3 +15,5 @@ Tutorials to help you get started with Nextstrain!
    tb_tutorial
    SARS-CoV-2/index
    narratives-how-to-write
+   nextclade-web-tutorial
+   nextclade-cli-tutorial
```
Comment (eharkins) [inline src/tutorials/index.rst:19]:
```suggestion
   nextclade-web
   nextclade-cli
```

File: src/learn/index.rst [hunk 8]
```diff
@@ -13,3 +13,4 @@ Learn about Nextstrain and related background information.
    About Nextstrain <about-nextstrain>
    interpret/index
    pathogens/index
+   Nextclade analysis <nextclade>
```
Comment (eharkins) [inline src/learn/index.rst:16]: See above comment, this doesn't work as is and so should be converted to a link or else the whole algorithm section needs to get fetched and included in the right directory
```suggestion
   Nextclade analysis <https://docs.nextstrain.org/projects/nextclade/en/latest/user/algorithm/index.html>
```

File: .gitignore [hunk 9]
```diff
@@ -19,3 +19,8 @@ src/guides/install/auspice-install.md
 src/guides/install/cli-install.md
 src/tutorials/narratives-how-to-write.md
 src/tutorials/SARS-CoV-2/steps/*.md
+src/learn/nextclade.md
+src/reference/nextalign-cli.md
+src/reference/nextclade-cli.md
+src/tutorials/nextclade-cli-tutorial.md
+src/tutorials/nextclade-web-tutorial.md
```
Comment (eharkins) [inline .gitignore:26]:
```suggestion
src/tutorials/nextclade-web.md
```

File: .gitignore [hunk 10]
```diff
@@ -19,3 +19,8 @@ src/guides/install/auspice-install.md
 src/guides/install/cli-install.md
 src/tutorials/narratives-how-to-write.md
 src/tutorials/SARS-CoV-2/steps/*.md
+src/learn/nextclade.md
```
Comment (eharkins) [inline .gitignore:22]: This doc doesn't exist. Maybe it used to? See below for comments on how to potentially include the algorithm docs.
```suggestion
```

**Review by eharkins — CHANGES_REQUESTED** [review]
Thanks for this! Some tweaks to links, etc needed but great to see the nextclade docs integrated with main nextstrain docs. Seems like this can move forward now that https://docs.nextstrain.org/projects/nextclade/en/latest/index.html is live.

**Comment by ivan-aksamentov** [comment]
Closing, because we moved Nextclade docs to a RTD subproject of the main docs, so files not need to be copied explicitly. We will add a link to subproject in a separate PR.
https://github.com/nextstrain/nextclade/tree/master/docs

**Pull request closed without merging**